src/components/picture-view-modal.vue

```diff
@@ -1,15 +1,22 @@
 <template>
- <div class="slidebox shown" :style="extraStyle" @dblclick="like" ref="slideBox">
-  <span class="like-icon" ref="liker">
-    <ion-icon :icon="likeIcon"  />
-  </span>
+  <div
+    class="slidebox shown"
+    :style="extraStyle"
+    @dblclick="like"
+    ref="slideBox"
+  >
+    <span class="like-icon" ref="liker">
+      <ion-icon :icon="likeIcon" />
+    </span>
 
-    <ion-img class="item-img" :src="imageUrl" />
-  
-   <div class="menu">
-    <router-link :to="teamLink">
-      <avatar size="4em" :profile="team" :name="teamName" />
-    </router-link>
+    <!-- <ion-img class="item-img" :id="imageUrl" v-if="imageUrl" :src="imageUrl" /> -->
+    <img class="item-img" :id="imageUrl" v-if="imageUrl" :src="imageUrl" />
+
+    <div class="menu">
```

Comment:
unrelated to this PR but this is wrong. Follow up issue: https://github.com/TeamFranka/affinity/issues/207